### PR TITLE
Sigloi: v0.5

### PR DESCRIPTION
**Pull request by AntoineRondelet**
Merging the content of v0.5 to master

### CODE REVIEW

File: scripts/test_zeth_cli_token
Comment (AntoineRondelet) [inline scripts/test_zeth_cli_token:7]: RUNNING (I'd also add surrounding `"`)

**Comment by AntoineRondelet** [comment]
This failing docker check is really annoying. I raised it in https://github.com/clearmatics/zeth/issues/258 already, but we'd need to fix it sooner rather then later because this is very annoying... I cloned a fresh version of the repo and built the docker images, then pushed them to the Hub manually so this is not a blocker for this PR to go through, but continuously having failing checks (especially on new "releases" PR) is very annoying.

File: CMakeLists.txt
Comment (dtebbs) [inline CMakeLists.txt:303]: indentation

**Comment by dtebbs** [comment]
> I cloned a fresh version of the repo and built the docker images, then pushed them to the Hub manually so this is not a blocker for this PR to go through

So is this expected to pass once when this is merged to master?
If not, can this be removed from the checks until it passes?

> but continuously having failing checks (especially on new "releases" PR) is very annoying.

Yes indeed.  I get a 404 when I try to see the details, so not sure whether it's something trivial or likely to keep happening.

**Review by dtebbs — APPROVED** [review]

**Comment by dtebbs** [comment]
LGTM when #279 is merged.

**Comment by AntoineRondelet** [comment]
> So is this expected to pass once when this is merged to master?
> If not, can this be removed from the checks until it passes?

I don't think so. I get a 404 when trying to figure out what's happening, and can't see the logs when I try to access them from the Hub directly, so tricky to debug and fix from this perspective...
We'd need support from DevOps to fix this (as per my issues #258 and #278) as it seems this auto build was configured with an account I don't have (and don't want) to have control over.

**Comment by AntoineRondelet** [comment]
All the checks have successfully be ran to the exception of the docker autobuild as per my comment above and the issues raised.
All the docker images have been built with the `0.5` tag and successfully pushed to the Hub.
Merging this PR.

**Pull request merged**
